lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedAccessControlConfig.java

```diff
@@ -56,4 +61,27 @@ public FileBasedAccessControlConfig setRefreshPeriod(Duration refreshPeriod)
         this.refreshPeriod = refreshPeriod;
         return this;
     }
+
+    @NotNull
+    public String getJsonPointer()
+    {
+        return jsonPointer;
+    }
+
+    @Config(SECURITY_JSON_POINTER)
+    @ConfigDescription("JSON pointer (RFC 6901) to mappings inside JSON config")
+    public FileBasedAccessControlConfig setJsonPointer(String jsonPointer)
+    {
+        this.jsonPointer = jsonPointer;
+        return this;
+    }
+
+    public SecurityConfigSource getSecurityConfigSource()
```

Comment:
We don't have to map to a enum - we can have a method saying whether its http or not.

Reply:
done